cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,84 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+import faker from 'faker';
+
+const homePage = new HomePageObject();
+const signInPage = new SignInPageObject();
+const settingsPage = new SettingsPageObject();
+const profilePage = new ProfilePageObject();
+
 describe('Settings page', () => {
-  before(() => {
+  let user;
+  let newUserName = faker.name.firstName().toLowerCase();
+  let newUserEmail = faker.internet.email().toLowerCase();
+  let newUserBio = faker.random.words();
+  let newUserPassword = faker.internet.password();
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });    
   });
 
   beforeEach(() => {
-
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
+    settingsPage.userNameField.clear().type(newUserName);
```

Comment:
move this inside settings page and name fillTheUserNameField()

Reply:
Fixed